frontend/src/components/commons/ContentContainer.jsx

```diff
@@ -22,7 +22,9 @@ const ContentContainer = props => {
       {!props.hideBanner && (
         <img role="presentation" src={mini_banner} alt="" style={styles.banner} />
       )}
-      <div style={styles.container}>{props.children}</div>
+      <div role="complementary" style={styles.container}>
```

Comment:
Aren't things in the content container normally primary?

Reply:
Removed this role, since it already have a main role.
See commit #https://github.com/code-for-canada/project-thundercat/pull/476/commits/093d17b0dba4ef54089f2aca1e361acaab56bf4d